imap_processing/ultra/l1b/ultra_l1b_culling.py

```diff
@@ -103,8 +103,8 @@ def flag_attitude(eventtimes_met: NDArray) -> tuple[NDArray, NDArray, NDArray, N
     spin_period = spin_df.loc[spin_df.spin_number.isin(spins), "spin_period_sec"]
     spin_starttime = spin_df.loc[spin_df.spin_number.isin(spins), "spin_start_time"]
     spin_rates = 60 / spin_period  # 60 seconds in a minute
-    indices = (spin_rates > UltraConstants.CULLING_RPM_MIN) & (
-        spin_rates < UltraConstants.CULLING_RPM_MAX
+    indices = (spin_rates < UltraConstants.CULLING_RPM_MIN) & (
+        spin_rates > UltraConstants.CULLING_RPM_MAX
```

Comment:
Won't this always produce an empty result? Do you want to logical or them together instead? Previously your indices were the "good" ones between the MIN and MAX, and now I think they are the AND of less than min and more than max which won't be hit?